### ISSUE TITLE
Link error

### ISSUE THREAD

**Issue by kochev** [MvvmCross/MvvmCross#1334]
error XA2006: Reference to metadata item 'System.Boolean Android.OS.BaseBundle::GetBoolean(System.String,System.Boolean)' (defined in 'MvvmCross.Droid.Shared, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null') from 'MvvmCross.Droid.Shared, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null' could not be resolved. ---> Mono.Cecil.ResolutionException: Failed to resolve System.Boolean Android.OS.BaseBundle::GetBoolean(System.String,System.Boolean)
   в Mono.Linker.Steps.MarkStep.MarkMethod(MethodReference reference)
   в Mono.Linker.Steps.MarkStep.MarkInstruction(Instruction instruction)
   в Mono.Linker.Steps.MarkStep.MarkMethodBody(MethodBody body)
   в Mono.Linker.Steps.MarkStep.ProcessMethod(MethodDefinition method)
   в Mono.Linker.Steps.MarkStep.ProcessQueue()
   в Mono.Linker.Steps.MarkStep.Process()
   в Mono.Linker.Steps.MarkStep.Process(LinkContext context)
   в Mono.Linker.Pipeline.Process(LinkContext context)
   в MonoDroid.Tuner.Linker.Process(LinkerOptions options, LinkContext& context)
   в Xamarin.Android.Tasks.LinkAssemblies.Execute()
   --- Конец трассировки внутреннего стека исключений ---
   в Xamarin.Android.Diagnostic.Error(Int32 code, Exception innerException, String message, Object[] args)
   в Xamarin.Android.Tasks.LinkAssemblies.Execute()
   в Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
   в Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()	IikoDeliveryMan.Droid			

### Steps to reproduce

1.Set "SDK Linking assembly"

2.Rebuild

### Configuration

**Version:** 4.1.6 (targetFramework="monoandroid50")

**Platform:**Android

VS 2015

**Comment by Cheesebaron** [comment]
I think you should make a Xamarin Bugzilla report and ask what is going on.

**Comment by kochev** [comment]
@Cheesebaron Why? Its mvvmcross libs dont linking..

**Comment by thefex** [comment]
I suppose that DLL was built with different Xamarin version (do you use alpha channel). @Cheesebaron can each release contain information about Xamarin build version ?

**Comment by Cheesebaron** [comment]
No it is built against stable

**Comment by thefex** [comment]
Are all MvvmCross dlls built with same Xamarin version?
Maybe @kochev uses alpha/beta

**Comment by Cheesebaron** [comment]
All of the ones released yesterday are built against the same version, yes.

**Comment by kochev** [comment]
@Cheesebaron, @thefex  I use stable Xamarin version. Now i updated to released yesterday packages, and set target framework       to API 23 and it worked.

**Comment by chauncat** [comment]
I experienced the same issue.  The MvvmCross dlls are build to version v6.0 for android.  I noticed the following output tab (My logging is set to verbose). 

`warning XA0105: The $(TargetFrameworkVersion) for MvvmCross.Binding.Droid.dll (v6.0) is greater than the $(TargetFrameworkVersion) for your project (v5.0).`

When I changed the projects to Android 6.0 I still received the linker error.  When I opened the project file in a text editor I noticed that <TargetFrameworkVersion> was defined twice.  Once in the global property group set to 6.0 and once in the Debug property group set to 5.0.  I removed the following line from the Debug property group and the issue went away.
 `<TargetFrameworkVersion>v5.0</TargetFrameworkVersion>`
